packages/sources/src/components/autosuggest/autosuggest.models.ts

```diff
@@ -1,6 +1,8 @@
 export interface Autosuggest {
   suggestions: AutosuggestSuggestion[];
   suggestOnFocus: boolean;
+  loading: boolean;
+  loadingLabel: string;
```

Comment:
Net zoals in de API van het Web Component is het `loadingLabel` attribuut optioneel. Dit model is leidend, dus dan moet dat hier ook optioneel zijn.

Het "Component model" is het contract tussen de functionaliteit en de techniek.